content/manuals/security/for-admins/single-sign-on/manage.md

```diff
@@ -54,24 +54,13 @@ aliases:
 
 ## Manage users
 
-{{< tabs >}}
-{{< tab name="Admin Console" >}}
-
 {{< include "admin-early-access.md" >}}
 
 {{% admin-sso-management-users product="admin" %}}
 
-{{< /tab >}}
-{{< tab name="Docker Hub" >}}
-
-{{% admin-sso-management-users product="hub" %}}
-
-{{< /tab >}}
-{{< /tabs >}}
-
```

Comment:
Does this mean that we can remove the `admin-sso-management-users` shortcode altogether? I think it was originally created to handle the hub/console conditional. But if we're no longer using the hub conditional, we should just inline the content for Admin console.

Reply:
that is a good point! moving forward in similar situations it would probably make most sense to remove the shortcode altogether :)